src/lib/components/chat/ChatInput.svelte

```diff
@@ -250,6 +252,27 @@
 						</label>
 					</HoverTooltip>
 				</form>
+				{#if mimeTypes.includes("image/*")}
+					<HoverTooltip label="Capture screenshot" position="top">
```

Comment:
positionning could be better: 
<img width="296" alt="image" src="https://github.com/user-attachments/assets/a04fbb15-4ee3-48a9-9dd5-4aac55b13e4b" />

Reply:
fixed